src/utilities/size/demo/block-size.twig

```diff
@@ -4,7 +4,7 @@
       full (in a 150px tall container)
     </div>
   </div>
-  <div style="height:150px;">
+  <div style="height:150px;" class="t-alternate">
```

Comment:
Unrelated change that didn't get included in the last PR that makes the block size utilities demo easier to understand.